docs/apidoc.md

```diff
@@ -143,3 +137,20 @@ Example
 In order for the endpoint to save data in the current session
 - the session name must start with the microservice name Example: @apiSuccess (Session) {String} microservice-account-id
 - technically to return the session, you return it as a custom header with the response
+
+## Ignore directory
+
+This is a work in progress, but now if you don't want a folder to be parsed by apidoc (not going to be shown in the docs), the folder must start with:
+
+```
+_kintoignore
+```
+
+## View auto generated Documentation
+
+One of the bonuses of documenting your code, you will be able to view the endpoints and there documentation in KintoHub
```

Comment:
We need to mention somewhere that @api and @apiName are required to even work with KintoHub.

Reply:
the beginning of the doc file i am saying what is required and what is not

Reply:
:+1: